### PR TITLE
Xmlapi/char/accountbalance

### PR DESCRIPTION
**Pull request by HelOWeen**
New endpoint added: /char/accountbalance

### CODE REVIEW

**Comment by Regner** [comment]
Sorry, I thought I commented on these pull requests already. Not sure whats going on with me lately. Was waiting for your responses on the comments I would have sworn that I had made. :(

File: docs/xmlapi/endpoints.md [hunk 1]
```diff
@@ -16,6 +308,53 @@ accessMask = 33554432
 ``/char/AccountBalance.xml.aspx``  
 
 * __Access mask:__ 1
+* __Paramaters:__
+   * __keyID__ (int)
```
Comment (Regner) [inline docs/xmlapi/endpoints.md]: 3 spaces instead of 4.

File: docs/xmlapi/endpoints.md [hunk 2]
```diff
@@ -16,6 +308,53 @@ accessMask = 33554432
...
+        </tr>
+        <tr>
+            <td>balance</td>
+            <td>decimal</td>
+            <td>Amount of ISK in this character's wallet</td>
+        </tr>
+    </tbody>
+</table>
+
+Notes
```
Comment (Regner) [inline docs/xmlapi/endpoints.md]: Missing a : at the end of this line.

File: docs/xmlapi/endpoints.md [hunk 3]
```diff
@@ -16,6 +308,53 @@ accessMask = 33554432
...
+* __accountKey:__ This is always 1000 for a character's account balance as opposed to a corporation, where accountKey ranges from 1000 to 1006 for the seven corporate wallet divisions.
```
Comment (Regner) [inline docs/xmlapi/endpoints.md]: There has to be a line break before lists for them to be rendered correctly.

**Comment by HelOWeen** [comment]
> In the future always fork your branches from the master. This PR has new changes and the changes from your previous PR as your branched from that branch instead of master. Does that make sense?

Agreed, makes sense.

**Comment by HelOWeen** [comment]
@Regner : I'd suggest you drop both pull requests #17 and #18 and I start over again by basing all branches off of master. My repo really looks ...erh ... "funny" now. Terrible mess which I need to sort out locally first. I need to grok this Github thingy first, I guess.

**Pull request closed without merging**